### PR TITLE
Html structure

### PR DESCRIPTION
**Pull request by augustinanders**
add the html structure and some first content

### CODE REVIEW

**Review by manukao — COMMENTED** [review]
lgtm

File: index.html [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+            <li>
+              <strong>02/2023 - current</strong> neuefische, web-dev trainee
+            </li>
+            <li>
+              <strong>09/2019 - 07/2022</strong> Berufliche Schule 6,
+              photography class
+            </li>
+          </ul>
+        </article>
+      </article>
```
Comment (FrankSiyli) [inline index.html:47]: double closing tags

File: index.html [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+      <article id="contact">
+        <h2>contact</h2>
+        <a href="">GitHub</a>
```
Comment (FrankSiyli) [inline index.html:50]: links missing for github and linkedin

**Review by FrankSiyli — CHANGES_REQUESTED** [review]
rest lgtm✌️

File: index.html [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    <title>Augustin Anders</title>
+  </head>
+  <body>
+    <header>
+      <h1>Augustin Anders</h1>
+      <nav>
+        <button type="button">portfolio</button>
+        <button type="button">about</button>
+        <button type="button">contact</button>
+      </nav>
```
Comment (Satttoshi) [inline index.html:16]: I think the buttons have no real navigation purpose at the moment. You might want to add href="#id" to create a working navigation.

**Review by Satttoshi — COMMENTED** [review]
Need some changes, which @FrankSiyli stated, after it should be LGTM ✌️

**Pull request merged**